src/views/Notifications/Home/_components/DailyDigest.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Grid, Icon } from '@trussworks/react-uswds';
+import { GetNotifications_currentUser_notifications_notifications_activity_metaData_DailyDigestCompleteActivityMeta_analyzedAudits as AnalyzedAuditsTypes } from 'gql/gen/types/GetNotifications';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import PageHeading from 'components/PageHeading';
+
+import {
+  pushValuesToChangesArray,
+  translatePlanSections,
+  TranslateStatusChange
+} from './_utils';
+
+const DailyDigest = ({
+  analyzedAudits
+}: {
+  analyzedAudits: AnalyzedAuditsTypes[];
+}) => {
+  const { t: notificationsT } = useTranslation('notifications');
+
+  return (
+    <Grid
+      desktop={{ col: 12 }}
+      className="border-1 border-base-lightest padding-x-3 padding-y-5"
+      data-testid="notification--daily-digest"
+    >
+      <PageHeading headingLevel="h2" className="margin-top-0 margin-bottom-4">
+        {notificationsT('index.dailyDigest.heading')}
+      </PageHeading>
+
+      {analyzedAudits.map(
+        ({
+          modelName,
+          modelPlanID,
+          changes,
+          changes: {
+            modelPlan,
+            documents,
+            crTdls,
+            planSections,
+            modelLeads,
+            planDiscussions
+          }
+        }) => {
+          const changesArray: any = [];
+          pushValuesToChangesArray(changes, changesArray);
+
+          const showFirstFiveChanges = changesArray.slice(0, 5);
+
+          return (
+            <div key={modelPlanID} className="margin-bottom-4">
+              <PageHeading
+                headingLevel="h3"
+                className="margin-top-0 margin-bottom-1"
+              >
+                {modelName}
+              </PageHeading>
+              <ul className="padding-left-205">
+                {modelPlan &&
+                  modelPlan.oldName &&
```

Comment:
This can be written using an optional chaining operator - `modelPlan?.oldName &&`